### DISCUSSION TITLE
Can't install react-typography nor gatsby-plugin-typography

### DISCUSSION THREAD

**Discussion by onni82** [gatsbyjs/gatsby#31051] [Help]
## Description

I tried running the command quoted in this step [✋ Install and configure gatsby-plugin-typography](https://www.gatsbyjs.com/docs/tutorial/part-three/#-install-and-configure-gatsby-plugin-typography).

All I receive are errors and I don't know what's causing this.

### Steps to reproduce

I ran: npm install gatsby-plugin-typography react-typography typography typography-theme-fairy-gates

I only received errors and received the suggestion to remove the node_modules foler and the package-lock.json file. I removed them and ran the command again. Didn't work. I tried installing each package with one command for each package, which led to typography and typography-theme-fairy-gates being installed, but gatsby-plugin-typography and react-typography couldn't be installed.

### Expected result

The packages should be installed, right?

### Actual result

Error codes:
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: gatsby-starter-hello-world@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!   peer react@"^16.9.0 || ^17.0.0" from gatsby-plugin-typography@3.3.0
npm ERR!   node_modules/gatsby-plugin-typography
npm ERR!     gatsby-plugin-typography@"*" from the root project
npm ERR!   2 more (gatsby, react-dom)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^0.14.0 || ^15.0.0 || ^16.0.0" from react-typography@0.16.19
npm ERR! node_modules/react-typography
npm ERR!   react-typography@"*" from the root project
npm ERR!   peer react-typography@"^0.16.1 || ^1.0.0-alpha.0" from gatsby-plugin-typography@3.3.0
npm ERR!   node_modules/gatsby-plugin-typography
npm ERR!     gatsby-plugin-typography@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\Onni\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Onni\AppData\Local\npm-cache\_logs\2021-04-24T12_09_29_947Z-debug.log

### Environment

  System:
    OS: Windows 10 10.0.19041
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
  Binaries:
    Node: 16.0.0 - C:\Program Files\nodejs\node.EXE
    npm: 7.10.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 90.0.4430.85
    Edge: Spartan (44.19041.906.0), Chromium (90.0.818.46)
  npmPackages:
    gatsby: ^3.3.1 => 3.3.1
  npmGlobalPackages:
    gatsby-cli: 3.3.0

**Comment by thijsv** [comment]
I have the same problem. Seems like a dependency issue.
Can resolve it by adding --legacy-peer-deps but don't think it is a sustainable solution.
